tests/app/services/test_process_request_json.py

```diff
@@ -162,21 +168,22 @@ def test_missing_field_in_transformation():
                 ]
             }
         },
-    ], TEXT_FIELDS_SET=set(['someField', 'otherField'])):
-        request = {
-            "otherField": ["wibble"],
-        }
+    ]
+    services_mapping.text_fields_set = set(['someField', 'otherField'])
+    request = {
+        "otherField": ["wibble"],
+    }
 
-        result = convert_request_json_into_index_json(request)
-        # really just checking this case doesn't throw!
+    result = convert_request_json_into_index_json(request)
+    # really just checking this case doesn't throw!
 
-        assert result == {
-            "otherField": ["wibble"],
-        }
+    assert result == {
```

Comment:
... less happy to start making actual changes to the tests in this PR. If you reckon these tests need work (I'm absolutely sure they could be improved) and it has value then we should be creating a separate ticket for that?